bin/configurer/utils/ccd-import-definition.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
I know it just copied code, but actually the name of that file is funny. A level above we do import-ccd-definition and then ccd-import-definition is like almost the same :D